src/posit/connect/external/aws.py

```diff
@@ -13,12 +13,34 @@
 
 def get_aws_credentials(client: Client, user_session_token: str) -> Dict[str, str]:
     """
-    Get AWS credentials using OAuth token exchange.
+    Get AWS credentials using OAuth token exchange for an AWS Viewer integration.
 
     According to RFC 8693, the access token must be a base64 encoded JSON object
```

Comment:
nit in these docstrings: I think base64 encoded should be hyphenated (base64-encoded). 

Obviously not remotely a big deal, just saw it as I was reading